torch/csrc/cuda/Module.cpp

```diff
@@ -909,6 +909,38 @@ PyObject* THCPModule_cudaGetSyncDebugMode(PyObject* self, PyObject* noargs) {
 static void registerCudaDeviceProperties(PyObject* module) {
   // Add _cudaDevicePropertires class to torch._C
   auto m = py::handle(module).cast<py::module>();
+  py::class_<CUuuid>(m, "_CUuuid")
+      .def_property_readonly(
+          "bytes", [](const CUuuid& uuid) { return py::bytes(uuid.bytes); })
+      .def("__repr__", [](const CUuuid& uuid) {
+        constexpr size_t size = 36 + 1;
```

Comment:
Can we read the size of cudaUUID_t here to reduce hard-coding?

Reply:
UUIDs are a 128-bit label.  CUDA and HIP store this as char[16].  For string representation, the code here expands this to 8-4-4-4-12 hex format, so it will be 32 ascii + 4 hyphens + 1 for the null byte.

If it helps I could add the above as a comment to indicate why the size is hard-coded?

Reply:
I guess that would be sizeof(cudaUUID_t) * 2 + 4 + 1 then? And with the comment to explain the 4 and 1. Sounds good!

Reply:
Done.